crates/db_schema/src/schema.rs

```diff
@@ -711,7 +711,7 @@ diesel::table! {
         id -> Int4,
         #[max_length = 200]
         name -> Varchar,
-        #[max_length = 512]
+        #[max_length = 2000]
```

Comment:
Can you use the `URL_MAX_LENGTH` that's defined in `validation.ts` instead of a magic number?

Reply:
For this file I can't, because its auto-generated.